internal/removals/remove.go

```diff
@@ -153,6 +153,16 @@ func GetRemainingMap(logger logr.Logger, adminClient fdbadminclient.AdminClient,
 		logger.Info("Exclusions to complete", "remainingServers", remaining)
 	}
 
+	if status.Cluster.RecoveryState.ActiveGenerations > 1 {
+		return nil, fmt.Errorf("too many active generations to run the exclude command to verify a process is fully excluded, current active generations: %d", status.Cluster.RecoveryState.ActiveGenerations)
+	}
+
+	// Make sure we use the exclude command again to verify that the processes are excluded based on the command.
```

Comment:
It appears there are a bunch of issues, as seen from the operator side, around exclude. Maybe we should summarize those issues and come up with a comprehensive solution(s) (from both server and operator sides) to address those issues?

Reply:
We have a plan to do this as a follow up.